test/unit/org/apache/cassandra/index/sai/cql/StorageAttachedIndexDDLTest.java

```diff
@@ -117,8 +163,8 @@ public void shouldFailCreationOnPartitionKey()
     {
         createTable("CREATE TABLE %s (id text PRIMARY KEY, val text)");
         assertThatThrownBy(() -> executeNet("CREATE CUSTOM INDEX ON %s(id) USING 'StorageAttachedIndex'"))
-                .isInstanceOf(InvalidQueryException.class)
-                .hasMessageContaining("Cannot create secondary index on the only partition key column id");
+        .isInstanceOf(InvalidQueryException.class)
+        .hasMessageContaining("Cannot create secondary index on the only partition key column id");
```

Comment:
nit: Might be useful to have the error message patterns as constants in `CreateIndexStatement` in case we ever have to tweak them.